quipucords/api/source/serializer.py

```diff
@@ -349,16 +379,16 @@ def validate_name(name):
 
     # pylint: disable=too-many-locals, too-many-branches, too-many-statements
     @staticmethod
-    def validate_hosts(hosts):
-        """Make sure the hosts list is present."""
-        hosts_list = json.loads(hosts)
-        if not isinstance(hosts_list, list):
+    def validate_ipaddr_list(hosts):
+        """Make sure the hosts list is present and has valid IP addresses."""
+        ipaddr_list = json.loads(hosts)
+        if not isinstance(ipaddr_list, list):
```

Comment:
This error message should be generalized a bit.  It indicates the problem is with host.